lib/frontend/app.js

```diff
@@ -108,6 +109,23 @@ module.exports = (appTokenGenerator) => {
       'Not Found': 404
     }
 
+    Sentry.withScope(async scope => {
+      scope.setTag('jiraHost', req.session.jiraHost || req.query.xdm_e)
+
+      if (req.session.githubToken) {
+        try {
+          const { data: { login } } = (await res.locals.github.users.get())
+          Sentry.setUser({ login })
+        } catch (_) {
+          console.log('unable to get user info')
```

Comment:
Is this something that should be logged using the probot logger, [similar to this](https://github.com/integrations/jira/blob/6239bf0176c1bfbf2a65548a4a073c455a9c8cc5/lib/jira/client/axios.js#L4)?

Reply:
Any preferences? I wrote it this way to explicitly swallow the error inside the `withScope` block. My thought here is we just want to attempt to get the user info if we can, but otherwise we don't want the rest of this block to fail.

Reply:
I like how you've added the user when it is available. It works as it stands. If you know what could go wrong, it may help to match on the error before swallowing or including the error in the log message.

In regards to _where_ to log, I lean toward using a logger over printing directly to STDOUT with `console.log`. It gives us a few extra details, which may prove useful.

I do have a follow-up: does everything printed to STDOUT make it to Loggly?

Reply:
> does everything printed to STDOUT make it to Loggly?

Yep. And we choose what goes to STDOUT based on the LOG_LEVEL env variable.

> If you know what could go wrong

Yeh that's part of the issue. It's just a catchall for any kind of problem getting data from the GitHub API. If it were a critical path I'd want to rescue various errors, but since this is just about decorating the error, I think we're ok here just moving on.

I do like the idea of using the real application logger. Let me see if I can get that in scope here.